libs/internal/include/launchdarkly/data_model/flag.hpp

```diff
@@ -0,0 +1,85 @@
+#pragma once
+
+#include <boost/json/value.hpp>
```

Comment:
Splitting these into groups would be nice.

Reply:
Clang format is doing something smart. But I guess not smart enough, I haven't paid attention to how exactly it is ordering.

Reply:
It organizes things within any existing groups. It just alphabetizes things. In lots of the other files I had split the groups, then it will help maintain them.

Reply:
It isn't strictly needed, but I think it makes life nice in the long-term.